src/styles/_grid.sass

```diff
@@ -141,11 +141,6 @@
           left: $perc
         
         $i: $i + 1
-      
-    
-  
-
 
 .row.full-height
-  min-height: 100vh
-
+  min-height: 100vh
```

Comment:
Good point. I checked http://caniuse.com/#feat=viewport-units and it looks like support is pretty good. If users are on anything older than can support viewport units...they're gonna have a ton of other problems on the app, so gonna keep this in for MVP.